packages/camera/camera/lib/camera.dart

```diff
@@ -6,4 +6,10 @@ export 'src/camera_controller.dart';
 export 'src/camera_image.dart';
 export 'src/camera_preview.dart';
 
-export 'package:camera_platform_interface/camera_platform_interface.dart';
+export 'package:camera_platform_interface/camera_platform_interface.dart'
+    show
+        CameraDescription,
+        CameraException,
+        CameraLensDirection,
+        ResolutionPreset,
+        XFile;
```

Comment:
Compare how much cleaner this is now with everything that is exported by the platform_interface:
https://pub.dev/documentation/camera_platform_interface/latest/camera_platform_interface/camera_platform_interface-library.html! Thanks!!